src/CommerceModel.BetterRetail/artifacts/OOE/BetterRetail/ImportEmailTemplatesActivity/Global/OrderCreated/Contents/Html/en-CA.html

```diff
@@ -295,10 +295,10 @@ <h2 style="font-size: 24px; margin-bottom: 0px;">
 										<tr>
 											<td colspan="2"></td>
 											<td colspan="2" style="text-transform: uppercase; line-height: 20px; text-align: left;">
-												<strong style="font-size: 18px;">Adjustment</strong>
+												<span style="text-transform: none;">Adjustment</span>
```

Comment:
do we need to do the same change for the next section (Total) ?

Reply:
No, that was my mistake actually, i wrongly used the total style on Adjustment, which i should not have

Reply:
![image](https://github.com/user-attachments/assets/d02c8ea6-08f8-4c57-9240-bf5f7ad50424)